### DISCUSSION TITLE
Roadmap for S2

### DISCUSSION THREAD

**Discussion by xingwanying** [antvis/S2#258] [Announcements]
### :building_construction: 基础架构

![image](https://user-images.githubusercontent.com/10885578/133040884-73801b9b-7a33-4bdc-9811-fca131de8f12.png)

### :triangular_flag_on_post: 里程碑

![image](https://user-images.githubusercontent.com/10885578/133039786-d513407c-22e6-4217-b7e5-a2f6dd950499.png)

**Comment by qubaomingg** [comment]
interaction 注册过程可以体现在图片上ma ?

**Reply by xingwanying** [reply]
注册过程属于流程图吧？太细节了会不会

**Comment by tobemaster56** [comment]
都已经准备发布v2，roadmap能否更新一下

文本自动换行
自适应列宽

像这些功能都是大家呼声比较高的，如果有明确的路线图，岂不是更好
